### ISSUE TITLE
Add 'asset' and 'path' 

### ISSUE THREAD

**Issue by eduardomart** [bolt/docs#645]
`paths` is being deprecated, per @CarsonF 
`asset` and `path` need to be documented in the cheatsheet
see his PR here:
https://github.com/bolt/bolt/pull/5836

**Comment by GawainLynch** [comment]
You mean like here https://github.com/bolt/docs/blob/release/3.0/docs/templating/templatetags.md

**Comment by bobdenotter** [comment]
That's the regular docs.. Cheatsheet needs updating, too. :-)

**Comment by GawainLynch** [comment]
Ah, I thought you were in the process of replacing that with something new.

**Comment by bobdenotter** [comment]
I think a lot of people use the cheatsheet. I don't think it'd be smart to replace/remove it.

**Comment by GawainLynch** [comment]
I just thought there was some Bob Magic (based on your discussion in the dev meeting last year) about something more applicable.

But if that's canned, so be it.

**Comment by bobdenotter** [comment]
I think we had some plans to work on contextual help, that could be linked to directly from Bolt, but that's additional to what we have. :-)

**Comment by GawainLynch** [comment]
Yes, just was looking over the few months of discussion about it, you were going to merge it into the main site proper (passed the URL move) and something something contextual something.

So I stand by my original point, but if we're keeping it for now, then yes needs updating.

**Comment by GawainLynch** [comment]
@bobdenotter assigning to you, posibly the most over worked person in the project currently … if anyone wants to step up and take this one on it would be greatly appreciated.
